be/src/exec/pipeline/exec_state_reporter.h

```diff
@@ -22,7 +22,7 @@ class ExecStateReporter {
                                                                     bool done);
     static Status report_exec_status(const TReportExecStatusParams& params, ExecEnv* exec_env, TNetworkAddress fe_addr);
     ExecStateReporter();
-    void submit(FragmentContext* fragment_ctx, const Status& status, bool done, bool clean);
+    std::unique_ptr<ThreadPoolToken> new_token() { return _thread_pool->new_token(ThreadPool::ExecutionMode::SERIAL); }
```

Comment:
Comment why must use SERIAL `ExecutionMode`?

Reply:
The report tasks submitted in the same query need to ensure the order of execution